docs/structure.md

```diff
@@ -36,28 +39,38 @@ More in-depth structure:
 │   └── tests.md
 ├── help.txt
 ├── images
+│   ├── cli.gif
+│   ├── coverage_report.PNG
 │   ├── ocean.gif
-│   ├── surf.gif
+│   ├── old_cli.gif
 │   ├── wave.png
 │   └── website.gif
+├── makefile
 ├── mkdocs.yml
+├── poetry.lock
 ├── pyproject.toml
 ├── README.md
-├── requirements.txt
 ├── src
 │   ├── api.py
 │   ├── art.py
 │   ├── cli.py
+│   ├── gpt.py
 │   ├── helper.py
 │   ├── __init__.py
 │   ├── __pycache__
 │   ├── send_email.py
 │   ├── server.py
+│   ├── settings.py
 │   ├── static
 │   └── templates
 ├── start_venv.sh
```

Comment:
With the introduction of Poetry, I think this file is no longer being used, so it might be okay to delete it!

Reply:
While it's not necessary now, if the number of project files increases in the future, modifying this file could become a hassle. So, ex.g it might be a good idea to add a simple script to `pre-commit` that automatically retrieves and reflects the directory structure :)

Reply:
> With the introduction of Poetry, I think `start_venv.sh` is no longer being used, so it might be okay to delete it!

Ah, forgot about that. Good call

Reply:
> While it's not necessary now, if the number of project files increases in the future, modifying this file could become a hassle. So, ex.g it might be a good idea to add a simple script to `pre-commit` that automatically retrieves and reflects the directory structure :)
> 
> 

How did I not think of this...good idea. I'll work on getting that going